### ISSUE TITLE
Can't parse some gear 3.1.3

### ISSUE THREAD

**Issue by dmrickey** [Lavaeolous/PF1-StatBlock-Converter-Module#392]
`Gear +1 ominous dagger, Cloak of Resistance +2, Headband of Alluring Charisma +2`

It wasn't able to find the dagger because it's magic, but I expected it to be able to find the cloak and the headband. The the gear was initially all lower case so I title-cased it and made sure it matched what was in the built in compendiums but it still didn't find them.

![image](https://user-images.githubusercontent.com/3664822/116641650-c7c57e80-a932-11eb-918a-8f9392b6a37c.png)

<details>
  <summary>Click to reveal stat block</summary>
  
```
ZERALISCE CR 10
XP 9,600
Female fetchling sorcerer 11
CG Medium outsider (native)
Init +2; Senses darkvision 120 ft., low-light vision; Perception +8
DEFENSE
AC 21, touch 13, flat-footed 18 (+4 armor, +2 Dex, +1 dodge, +4 shield)
hp 85 (11d6+33)
Fort +7, Ref +7, Will +9
Defensive Abilities shadow blending; Resist cold 5,
electricity 5
OFFENSE
Speed 30 ft.
Melee +1 ominous dagger +6 (1d4+1/19–20)
Fetchling Spell-Like Abilities (CL 11th; concentration +16)
1/day—disguise self, shadow walk (self only)
Sorcerer Spell-Like Abilities (CL 11th; concentration +16)
8/day—shadowstrike (1d4+5 nonlethal)
Sorcerer Spells Known (CL 11th; concentration +16)
5th (5/day)—cloudkill (DC 20), lightning arcUM (DC 20), shadow evocation (DC 20)
4th (7/day)—dimension door, fire shield, secure shelter, shadow conjuration
3rd (7/day)—daylight, deeper darkness, fireball (DC 18), rage, tongues
2nd (7/day)—darkness, darkvision, levitate, mirror image, scorching ray, web (DC 17)
1st (8/day)—burning hands (DC 17), comprehend languages, mage armor, magic missile, ray of enfeeblement (DC 16), shield
0 (at will)—acid splash, bleed (DC 15), dancing lights, detect magic, flare (DC 16), mage hand, message, mending, prestidigitation
Bloodline shadow
TACTICS
Before Combat Zeralisce maintains extended mage armor at all times. She also casts extended shield before venturing into a dangerous area. The bonuses to her AC are included above, but remember that when the PCs first encounter her, these effects have long since expired and she hasn’t been able to re-cast the spells.
During Combat Zeralisce makes her spells available to the PCs if they travel with her; several of them she’s perfected to aid in her role as a guide for off-plane patrons. She offers darkvision to PCs who can’t see in the dark, daylight to do what she can to offset the plane’s gloom, secure shelter if a safe place to rest is required, recovering as needed. In combat, she uses her shadow evocation and shadow conjuration spells to create effects tailored specifically for the situation, but she is always ready to extract allies in danger by rushing in and then casting dimension door to help them escape.
Morale If the PCs rescue Zeralisce, she will fight to the death to aid and protect them. But on her own, she flees if reduced below 20 hit points.
STATISTICS
Str 10, Dex 15, Con 14, Int 12, Wis 6, Cha 21
Base Atk +5; CMB +5; CMD 18
Feats Combat Casting, Dodge, Empower Spell, Eschew
Materials, Extend Spell, Iron Will, Silent Spell, Spell Focus (evocation)
Skills Diplomacy +8, Fly +8, Intimidate +6, Perception +8,
Spellcraft +12, Stealth +4, Use Magic Device +17
Languages Common, Shadowtongue
SQ shadow well
Gear +1 ominous dagger, Cloak of Resistance +2, Headband of Alluring Charisma +2
```  
</details>

**Comment by Lavaeolous** [comment]
That's on the roadmap with issue #371 as sbc currently only looks for direct matches.

**Comment by Lavaeolous** [comment]
These issues should be fixed in 3.2.4. Feel free to let me know if it's working, then i can close this issue.

**Comment by Lavaeolous** [comment]
3.1.4 is now online
